administrator/modules/mod_loginsupport/services/provider.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  mod_loginsupport
+ *
+ * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
```

Comment:
```suggestion
\defined('_JEXEC') or die;
```
Fix PHPCS failing in Drone.

Reply:
@richard67  Of course I can fix it, but I am confused, how to do it correctly. In different PR I did it this way - with \defined , but I was asked to remove \ from native functions in non-namespaced php files - https://github.com/joomla/joomla-cms/pull/42791#discussion_r1487197285, https://github.com/joomla/joomla-cms/pull/42791#discussion_r1487463780 .

@richard67, @Quy and @sandewt , should I add \ in non-namespaced defines, so build will not fail, or remove, but build will fail?

Reply:
Files **provider.php**:

Joomla 4.x
`defined('_JEXEC') or die;`

Joomla 5.x
`\defined('_JEXEC') or die;`

I was not aware of this difference.

To be consistent, I would here (in J5) use the code `\defined('_JEXEC') or die;`
However, I don't know the reason.

Reply:
@joomlaweby In the 5.0-dev branch (and so also 5.1-dev and so on) there was made a change, see PR #41427 . I remember it has been discussed in the maintainer team and it was decided to use the backslash in general, also for non namespaced files. So I would rely on the check in drone regarding this.

Reply:
OK, great! Thanks for explanation! :-) Now it is fixed